source/common/network/dns_resolver/dns_factory.h

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+#include "envoy/api/api.h"
+#include "envoy/config/bootstrap/v3/bootstrap.pb.h"
+#include "envoy/config/cluster/v3/cluster.pb.h"
+#include "envoy/event/dispatcher.h"
+#include "envoy/extensions/common/dynamic_forward_proxy/v3/dns_cache.pb.h"
+#include "envoy/extensions/filters/udp/dns_filter/v3alpha/dns_filter.pb.h"
+#include "envoy/extensions/network/dns_resolver/apple/v3/apple_dns_resolver.pb.h"
+#include "envoy/extensions/network/dns_resolver/cares/v3/cares_dns_resolver.pb.h"
+#include "envoy/network/dns.h"
+
+#include "source/common/config/utility.h"
+#include "source/common/runtime/runtime_features.h"
+
+namespace Envoy {
+namespace Network {
+
+const std::string cares_dns_resolver = "envoy.dns_resolver.cares";
+const std::string apple_dns_resolver = "envoy.dns_resolver.apple";
```

Comment:
naming convention for const is `AppleDnsResolver`, etc

Reply:
ACK